src/inc/staticcontract.h

```diff
@@ -237,7 +237,11 @@ namespace StaticContract
 {
     struct ScanThrowMarkerStandard
     {
+#if !defined(PLATFORM_UNIX)
         __declspec(noinline) ScanThrowMarkerStandard()
```

Comment:
You can use the NOINLINE macro which is defined like this:
#ifdef _MSC_VER
#define NOINLINE __declspec(noinline)
#elif defined __GNUC__
#define NOINLINE __attribute__ ((noinline))
#else
#define NOINLINE
#endif